### PR TITLE
feat(helm): add helm documentation support

### PR DESCRIPTION
**Pull request by tiagomichaelsousa**
## Description

This extension will allow to easily go through the Helm docs directly from Raycast. 
It will also allow searching within the documentation with algolia support and integration.
While developing the extension I tried to followed the same conventions as:

- [algolia](https://github.com/tiagomichaelsousa/extensions/tree/main/extensions/algolia)
- [laravel-docs](https://github.com/tiagomichaelsousa/extensions/tree/main/extensions/laravel-docs)
- [spatie-documentation](https://github.com/tiagomichaelsousa/extensions/tree/main/extensions/spatie-documentation)

## Type of change

- New extension

### Setup screenshot
setup

### List Documentation
list

### Search in Documentation with Algolia
search

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/helm-docs/src/documentation/v3.8.0.json [hunk 1]
```diff
@@ -0,0 +1,484 @@
+{
+  "Introduction": [
+    {
+      "title": "Quickstart Guide",
+      "url": "https://helm.sh/docs/intro/quickstart"
```
Comment (pernielsentikaer) [inline extensions/helm-docs/src/documentation/v3.8.0.json:5]:
```suggestion
      "url": "https://helm.sh/docs/intro/quickstart/"
```
NIT 😄

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @tiagomichaelsousa 🧇 

Awesome extension, thanks for your contribution 🔥 

Can you also provide a light icon? Otherwise, this won't look good with dark theme.

You can find [more info here](https://developers.raycast.com/basics/prepare-an-extension-for-store#extension-icon).

Besides that, well done 💪 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by pernielsentikaer** [comment]
@tiagomichaelsousa I just checked here: https://github.com/cncf/artwork/tree/master/projects/helm/icon – having it white for dark appearance could solve it, or?

**Comment by tiagomichaelsousa** [comment]
Having it white for the dark appearance would result in something like this (bellow) but if we can change the icon based on the theme we will still have a problem with the icon that is defined for the command in the `package.json`. 

Screenshot 2022-02-11 at 12 17 20

I tried to create an icon based on the [Extensions Icon Template](https://www.figma.com/community/file/1030764827259035122) and it resulted in something like this which i think it's the best solution as it works both for raycast light and dark team 😎  wdyt ?

Screenshot 2022-02-11 at 12 16 58 Screenshot 2022-02-11 at 12 24 20

**Comment by pernielsentikaer** [comment]
Hi @tiagomichaelsousa 🔥 🔥 

Did you know you can add an icon for darkmode?

bgO6KgQQGcKMb4Ok

G1vRayvWgUGAiMQY

Just add @dark as you can see in the screenshots 🔥

File: extensions/helm-docs/package.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "helm-docs",
+  "title": "Helm Documentation",
+  "description": "Helm documentation",
```
Comment (pernielsentikaer) [inline extensions/helm-docs/package.json:5]:
```suggestion
  "description": "Quickly search Helm documentation",
```

**Review by pernielsentikaer — APPROVED** [review]
Haha, @tiagomichaelsousa! 👋 

with Raycast you always have to expect the unexpected 😄 

I'll share your kudos with the team, I'm very glad that you like it 💪

**Comment by tiagomichaelsousa** [comment]
Let's see if another idea cross my mind so that I can contribute again

Thank you @pernielsentikaer 🙏

**Pull request merged**